pkg/cdi/cdi.go

```diff
@@ -80,7 +75,14 @@ func (c *impl) CreateCDISpecForPool(resourcePrefix string, rPool types.ResourceP
 		cdiSpec.Devices = append(cdiSpec.Devices, device)
 	}
 
-	err = cdi.GetRegistry().SpecDB().WriteSpec(&cdiSpec, cdiSpecPrefix+resourcePrefix)
+	name, err := cdi.GenerateNameForSpec(&cdiSpec)
+	if err != nil {
+		glog.Errorf("CreateCDISpecForPool(): can not generate name: %v", err)
```

Comment:
```suggestion
		glog.Errorf("GenerateNameForSpec(): can not generate name: %v", err)
```

should it be like this?